Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Sysconf");
```

Comment:
Renaming any setting will lead to setting files being incompatible unless we implement some sort of migration. It might not be worth it.

There aren't really many settings that apply to the GameCube but not to the Wii. The only thing I can think of that for sure doesn't exist on a Wii is the SP1 slot. Memory cards can also be seen as a GameCube thing, although they technically work just as fine on a Wii - it's just that Wii games almost never make use of them. So even if we didn't have to worry about compatibility, I'm not sure if there's much we could put in a GameCube section.